rpa-manager/modules/ROOT/pages/processautomation-prepare-deployment-credential.adoc

```diff
@@ -0,0 +1,179 @@
+
+* xref:processautomation-deploy.adoc#test-configuration-link-globals[Linkable Activity Parameters in Test Configurations]
+* xref:processautomation-deploy.adoc#production-configuration-link-globals[Linkable Activity Parameters in Production Configurations]
+
+Editing or deleting a credential affects all configurations that use the credential.
+
+Check the usage of a credential before changing or deleting it.
+
+== Before You Begin
+
+* Ask an administrator to xref:usermanagement-manage.adoc#assign-privileges-to-a-user[assign you the needed privileges]:
++
+[cols="1,2"]
+|===
+|*Action* |*Required Privileges*
+
+|Open the *Process Automation* module
+|Process Automation Open
+
+|Create and change global variables and credentials that can be used in the test phase
+|Global Variables Create for Test phase, Global Variables Edit for Test phase
+
+|Create and change global variables and credentials that can be used in the production phase
+|Global Variables Create for Production phase, Global Variables Edit for Production phase
+
+|Delete global variables and credentials
+|Global Variables Delete
+
+|===
+* If you have access to an Amazon Web Services (AWS) account and want to create AWS credentials for xref:rpa-builder::toolbox-aws-document-processing.adoc[RPA Builder - Document Processing with AWS], follow the instructions in https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html[AWS configuration basics^].
+* If you have access to a Microsoft Azure account and want to create OAuth credentials for xref:rpa-builder::toolbox-mail-operations-mail-session-outlook-with-oauth.adoc[RPA Builder - Mail Session (Outlook with OAuth)], create an application for Outlook, https://learn.microsoft.com/en-us/exchange/client-developer/exchange-web-services/how-to-authenticate-an-ews-application-by-using-oauth[authenticate an EWS application by using OAuth^], and https://learn.microsoft.com/en-us/exchange/client-developer/legacy-protocols/how-to-authenticate-an-imap-pop-smtp-application-by-using-oauth[authenticate an IMAP, POP, or SMTP connection using OAuth^].
+* If you want to use RPA Builder to xref:rpa-builder::toolbox-variable-handling-credentials-for-oauth.adoc#generating-an-oauth-refresh-token[generate a refresh token for your OAuth credentials], ask an administrator to xref:usermanagement-manage.adoc#assign-users-rpa-builder-access[assign you RPA Builder access].
+
+== Create a Credential
+
+Create credentials with which a bot can log in to an external application during process runs.
+
+To create a credential:
+
+. Open the *Credential Pool* view of the *Process Automation* module.
+. Click *Create*.
+. [[form-create-credential]] Complete the *Create a New Credential* form:
+* [[form-create-credential-useraffiliation]] *User affiliation*
++
+Click to choose users that may view, edit, and use the credential. Add yourself so that you can also view, edit, and use the credential.
```

Comment:
```suggestion
Click to choose users who can view, edit, and use the credential. Add yourself so that you can also view, edit, and use the credential.
```

Reply:
Style says to use select in this case:
https://confluence.internal.salesforce.com/pages/viewpage.action?spaceKey=MTDT&title=MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22select,%22%22choose%22

Reply:
This is an permission/security feature. That's why I explcitly wrote 'that may'. Is that not necessary?

Reply:
Our style says to avoid modal words, but also says you can use it for permission. 

I think maybe make it explicit. Something like:
Select users to allow to view, edit, and use the credential.

It's up to you.